osdep/path-unix.c

```diff
@@ -40,25 +40,25 @@ static void path_init(void)
     char *xdg_state = getenv("XDG_STATE_HOME");
 
     bool err = false;
-    if (xdg_config && xdg_config[0]) {
+    if (xdg_config && xdg_config[0] == '/') {
         err = err || MKPATH(mpv_home, "%s/mpv", xdg_config);
-    } else if (home && home[0]) {
+    } else if (home && home[0] == '/') {
         err = err || MKPATH(mpv_home, "%s/.config/mpv", home);
     }
 
     // Maintain compatibility with old ~/.mpv
-    if (home && home[0])
+    if (home && home[0] == '/')
```

Comment:
This doesn't look right. `$HOME` is not part of XDG base dirs.

Same with all the other `home[0] == '/'` cases.